### PR TITLE
Enable format/inject plugin helper for Fluentd v0.14

### LINKED ISSUE #48: output_format , just simple
Now I have a problem,
> format whole data as serialized JSON, single attribute or separated multi attributes
or LTSV, labeled-TSV (see http://ltsv.org/ )

However,I just want raw data ,like input equal output
neither ``` {“message”:"......"}``` or ```message : .....```

### PR DESCRIPTION
**Pull request by tagomoris**
Fixes #48.

This change does:
* remove config-placehodler and plaintextformatter mixins
* enable format/inject plugin helper to control formatting in v0.14 standard way

This plugin will be ready for Fluentd v0.14 after merging this patch.

### CODE REVIEW

**Comment by tagomoris** [comment]
@repeatedly Could you review this code?

File: fluent-plugin-webhdfs.gemspec [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 Gem::Specification.new do |gem|
   gem.name          = "fluent-plugin-webhdfs"
-  gem.version       = "0.5.3"
+  gem.version       = "0.6.0rc1"
```
Comment (repeatedly) [inline fluent-plugin-webhdfs.gemspec]: If you update code to work with only v0.14, upgrade major version.

http://docs.fluentd.org/articles/plugin-development#plugin-versioning-policy

**Comment by repeatedly** [comment]
I don't know the plaintextformatter but others are good for me.

File: fluent-plugin-webhdfs.gemspec (hunk 1 above)
Comment (tagomoris) [inline fluent-plugin-webhdfs.gemspec]: Okay.

**Comment by tagomoris** [comment]
Let me merge this, and then fix the version number to `1.0.0` and release it.

File: lib/fluent/plugin/out_webhdfs.rb [hunk 2]
```diff
@@ -247,13 +264,52 @@ def send_data(path, data)
...
   def generate_path(chunk)
     hdfs_path = if @append
                   extract_placeholders(@path, chunk.metadata)
                 else
                   extract_placeholders(@path, chunk.metadata).gsub(CHUNK_ID_PLACE_HOLDER, dump_unique_id(chunk.unique_id))
                 end
     hdfs_path = "#{hdfs_path}#{@compressor.ext}"
+    if @replace_random_uuid
+      uuid_random = SecureRandom.uuid
+      hdfs_path.gsub!('%{uuid}', uuid_random).gsub!('%{uuid_flush}', uuid_random)
```
Comment (repeatedly) [inline lib/fluent/plugin/out_webhdfs.rb]: Is `gsub!` chaining safe?
`gsub!` returns `nil` when substitution doesn't happen.

**Pull request merged**